test-runner-poc/package.json

```diff
@@ -4,9 +4,14 @@
   "description": "",
```

Comment:
Is this still being run like a subpackage? We should probably at some point move to this sharing the main project's `package.json`. @mbroadst @kvwalker thoughts?

Reply:
I agree, but can we leave that for a different PR?